lib/src/builders/method.dart

```diff
@@ -18,33 +18,57 @@ import 'package:code_builder/src/builders/type.dart';
 import 'package:code_builder/src/tokens.dart';
 
 /// Short-hand for `new ConstructorBuilder(...)`.
-ConstructorBuilder constructor(
-    [Iterable<ValidConstructorMember> members = const []]) {
+ConstructorBuilder constructor([
+  Iterable<ValidConstructorMember> members = const [],
+]) {
   return _constructorImpl(members: members);
 }
 
 /// Short-hand for `new ConstructorBuilder(name)`.
-ConstructorBuilder constructorNamed(String name,
-    [Iterable<ValidConstructorMember> members = const []]) {
+ConstructorBuilder constructorNamed(
+  String name, [
+  Iterable<ValidConstructorMember> members = const [],
+]) {
   return _constructorImpl(name: name, members: members);
 }
 
+/// Various types of modifiers for methods.
+class MethodModifier implements ValidMethodMember {
```

Comment:
Are things like `static` and `final` considered modifiers? I'm also wondering if `package:meta` should be supported out-of-the-box (as a separate feature request probably), for things like `@protected` and `@mustCallSuper`.